### PR TITLE
[LDAP] add new option implemented in php 7.1

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | yes 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets |  #18448  
| License       | MIT
| Doc PR        | none 

Add new options to the LDAP connection class.

### CODE REVIEW

**Comment by Simperfit** [comment]
I've taken this files as a source: https://github.com/delphij/openldap/blob/master/include/ldap.h

File: src/Symfony/Component/Ldap/Adapter/ExtLdap/ConnectionOptions.php [hunk 1]
```diff
@@ -23,6 +23,9 @@
  */
 final class ConnectionOptions
 {
+    const X_TLS_CRL_NONE = 0;
+    const X_TLS_CRL_PEER = 1;
+    const X_TLS_CRL_ALL = 2;
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Ldap/Adapter/ExtLdap/ConnectionOptions.php]: Should use 0x... and be sorted with other X_* consts to me.

Comment (Simperfit) [reply src/Symfony/Component/Ldap/Adapter/ExtLdap/ConnectionOptions.php]: done

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Simperfit.

**Pull request merged**